internal/provider/resource_instance.go

```diff
@@ -343,13 +355,14 @@ func (r *instanceResource) Create(ctx context.Context, req resource.CreateReques
 	params := oxide.InstanceCreateParams{
 		Project: oxide.NameOrId(plan.ProjectID.ValueString()),
 		Body: &oxide.InstanceCreate{
-			Description: plan.Description.ValueString(),
-			Name:        oxide.Name(plan.Name.ValueString()),
-			Hostname:    oxide.Hostname(plan.HostName.ValueString()),
-			Memory:      oxide.ByteCount(plan.Memory.ValueInt64()),
-			Ncpus:       oxide.InstanceCpuCount(plan.NCPUs.ValueInt64()),
-			Start:       plan.StartOnCreate.ValueBoolPointer(),
-			UserData:    plan.UserData.ValueString(),
+			AutoRestartPolicy: oxide.InstanceAutoRestartPolicy(plan.AutoRestartPolicy.ValueString()),
```

Comment:
There's no Go SDK change needed here? I thought this was made `nullable` upstream so we need to marshal this to `null` instead of omit it.

Reply:
Updated the sdk and pulled in the latest version.